org.eclipse.jdt.ui.tests/pom.xml

```diff
@@ -39,7 +39,8 @@
           <useUIThread>true</useUIThread>
           <includes>
             <include>org/eclipse/jdt/ui/tests/AutomatedSuite.class</include>
-            <!-- <include>org/eclipse/jdt/ui/tests/LeakTestSuite.class</include> -->
+            <include>org/eclipse/jdt/internal/common/JUnit5TestSuite.class</include>
```

Comment:
There is no JUnit5TestSuite. If you plan to add it, please add it to this plugin.

Reply:
> There is no JUnit5TestSuite. If you plan to add it, please add it to this plugin.

It got lost somehow in rebase.
Can you have a look at 

https://github.com/eclipse-jdt/eclipse.jdt.ui/pull/129

Where I added it again why it is not executed?